src/tests/utils/fixtures/flask_fixture.py

```diff
@@ -0,0 +1,20 @@
+from src.tests.utils.fixtures.oauth2_fixture import build_user, build_client
```

Comment:
is this a good name `flask_fixture`?

could you think of a better naming for this file?

Reply:
Done. I renamed to `endpoint_request_fixture`